config/crds/migration.openshift.io_directvolumemigrationprogresses.yaml

```diff
@@ -35,6 +35,7 @@ spec:
     shortNames:
     - dvmp
     singular: directvolumemigrationprogress
+  preserveUnknownFields: false
```

Comment:
Worried about this, if we have fields not defined here this will break things. Have we run a smoke test w/ these CRD changes?

Reply:
@shawn-hurley I tested the migrations manually, was working fine and not breaking anything, I pinged forks around with the same concern but the opinion was that it shouldn't cause trouble.

Reply:
+1

Reply:
Jaydip just mentioned to me that this is needed because the `description` field is apparently unknown, and this is how `oc explain` is able to work.